### ISSUE TITLE
Moons dataset looks weird

### ISSUE THREAD

**Issue by kennysong** [distillpub/post--bayesian-optimization#16]
Nice article! The image of the moons dataset at the end of the article looks a bit strange since the points are heavily overlapping and not in a moon shape. 

<img width="704" alt="Screen Shot 2020-05-06 at 1 35 34 PM" src="https://user-images.githubusercontent.com/2199875/81138868-793f0780-8f9e-11ea-8b6e-9635d438292e.png">

Is it the correct image?

**Comment by apoorvagnihotri** [comment]
Yes, it is the correct image, though we have used a variance of 1. You can find the code used for replicating the above image [here](https://github.com/distillpub/post--bayesian-optimization/blob/master/public/images/main.ipynb).

**Comment by kennysong** [comment]
Thanks for the reference! Was the reason for the high noise just to make the task harder and spread out the performance of different hyperparameter sets?

**Comment by apoorvagnihotri** [comment]
Hi @kennysong. To be honest, we didn't consider varying the noise. But I think your idea should be valid in this case.

I did some experiments with low noise and Bayesian Optimization (BO) is performing better. PFA, moon's data results on SVM.

New, simple moon’s dataset. 
![](https://i.imgur.com/fJln2nm.png)

New accuracies on using an SVM.
![](https://i.imgur.com/aQbb9eM.png)

New accuracies on using an RF.
![](https://i.imgur.com/TwRdy5Q.png)

Our task is much difficult--because of using a very high noise in dataset--in the original article. Therefore, it was difficult for the BO to perform. As seen here, reducing the complexity helped BO achieve better results.

Thanks a lot for pointing this out. I’ll keep this issue open as I think this is an important point. :)

You can have a look at the ipynb used [here](https://gist.github.com/apoorvagnihotri/75ae8f799b1ed0924803daf1f5b00728).

**Comment by kennysong** [comment]
Oh cool! It does look like the original harder task spread out the performance of different algorithms more. With less noise, all approaches seem to do better and converge to the global optimum. 

Thanks for following up!
